demos/projects/ESPRESSIF/esp32/components/azure-sdk-for-c/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+set(ROOT_PATH
+    ${CMAKE_CURRENT_LIST_DIR}/../../../../../..
+)
+
+set(AZURE_IOT_MIDDLEWARE_FREERTOS
```

Comment:
>AZURE_IOT_MIDDLEWARE_FREERTOS [](http://example.com/codeflow?start=4&length=29)

much better to name is component freertos-middleware and make embedded sdk part of it.

Reply:
This is just an internal variable used by the cmakelists. The name matches the directory name, I would keep it as is.